packages/components/src/components/hds/code-block/index.ts

```diff
@@ -90,6 +90,7 @@ export default class HdsCodeBlock extends Component<HdsCodeBlockSignature> {
     );
 
     if (Prism?.plugins?.['NormalizeWhitespace']) {
+      // eslint-disable-next-line @typescript-eslint/no-unsafe-return, @typescript-eslint/no-unsafe-call, @typescript-eslint/no-unsafe-member-access
```

Comment:
Is there a ticket already for us to address these eslint-disables later?

Reply:
We have [an entire epic](https://hashicorp.atlassian.net/browse/HDS-4656) – can break these new additions into more actionable tasks